### DISCUSSION TITLE
revalidate with notion api does not work when deployed on vercel

### DISCUSSION THREAD

**Discussion by ichn-hu** [vercel/community#385] [Help] [answered]
Hi!

I am using vercel to deploy my nextjs blog that uses notion as CMS, and I used `revalidate: 10` in `getStaticProps` for refreshing content, although it worked amazingly well under development, I found it not working when deployed to vercel.

I was guessing it is because I migrated from next 11.1.2 to next 12.0.10, however after I revert the change, it is still not working.

Since I also used `notion-sdk-js`, https://github.com/vercel/community/discussions/371 might be related, however, I can't see runtime logs in the dashboard (only function logs, but I do not yet use any function, it load with runtime logs and flushed quickly and turned into a blank function log).

Is runtime log missing a bug for vercel? I want it back to be able to debug runtime bugs, I understand vercel does not store log for me, but at least I want it in real-time.

![image](https://user-images.githubusercontent.com/29735669/155039423-33f67c3d-e98e-419e-a19d-3c6dcb51ca20.png)

![image](https://user-images.githubusercontent.com/29735669/155039440-f21fff8e-67ca-4f58-a0c3-13ef846f1654.png)

if you want a deployment for further investigation, https://vercel.com/ichn-hu/notion-blog/uDYGZdmKrC3ieNtBciTLaujm7FJa

**Comment by ichn-hu** [comment, marked as answer]
@samsisle sorry to bother but I don't know whom to contact, would you please take a look or refer someone else to take a look? I am happy to respond to any investigation questions.

**Reply by samcx** [reply]
@ichn-hu Is your Environment Variables set correctly? Seems like your ISR page is not building properly in Production.

**Reply by ichn-hu** [reply]
> @ichn-hu Is your Environment Variables set correctly? Seems like your ISR page is not building properly in Production.

I saw runtime logs now, and it seems there is some problem in my code thanks to the error log.

**Reply by samcx** [reply]
I'm glad you were able to find the error!
